gensim/summarization/summarizer.py

```diff
@@ -322,15 +322,16 @@ def _build_hasheable_corpus(corpus):
 
 
 def summarize_corpus(corpus, ratio=0.2):
-    """Get a list of the most important documents of a corpus using a variation of the TextRank algorithm [1]_.
-     Used as helper for summarize :func:`~gensim.summarization.summarizer.summarizer`
+    """Get a list of the most important documents of a corpus using a
+    variation of the TextRank algorithm [1].
```

Comment:
probably, you can use text identifiers for it, for example `[some-paper]`

Reply:
AFAIK this bug is connected to trailing underscore in `[reference_name]_`. I'll try to find a better workaround.

Reply:
@anotherbugmaster add links to comment for this PR about this bug, I'll try to find workaround too

Reply:
Done!